doc/source/tutorials/instrument.rst

```diff
@@ -131,3 +131,20 @@ Let's see a minimal example:
             # also the results are, in qibolab, specific objects that need some kind
             # of conversion. Refer to the results section in the documentation.
             return results
+
+
+Dummy platform
+^^^^^^^^^^^^^^
+
+In addition to the real instruments presented above, Qibolab provides the :class:`qibolab.instruments.DummyInstrument`.
+This instrument represents a controller that returns random numbers of the proper shape when executing any pulse sequence.
+This instrument is also part of the dummy platform which is defined in :py:mod:`qibolab._core.dummy` and can be initialized as
+
+.. testcode::  python
+
+    from qibolab import create_platform
+
+    platform = create_platform("dummy")
+
+This platform is equivalent to real platforms in terms of attributes and functions, but returns just random numbers.
+It is useful for testing parts of the code that do not necessarily require access to an actual quantum hardware platform.
```

Comment:
This paragraph seems to me to fit better in the `main-documention/drivers.rst` where we mainly talk about existing instrument code and `DummyInstrument` is one of these. In this tutorial we are mainly talking about adding new instruments, so dummy doesn't seem much related but I am fine with it in any place.

Reply:
This is fine for me. In Qibocal we already switch to our own `dummy` called `mock` to disentangle qibocal from qibolab.